migrations/V1__Add_System_Intake.sql

```diff
@@ -0,0 +1,23 @@
+create table system_intake (
+    id uuid PRIMARY KEY not null,
+    eua_user_id text not null,
+    requester text,
+    component text,
+    business_owner text,
+    business_owner_component text,
+    product_manager text,
+    product_manager_component text,
+    has_isso bool,
```

Comment:
Is this a similar thing to the trb_collaborator, where we don't need both the boolean has_isso and the text isso? Or is it different somehow?